setup.py

```diff
@@ -20,6 +20,7 @@
     "rtree>=0.9.7",
     "psutil>=5.8.0",
     "arbor>=0.5",
+    "pynrrd>=0.4"
```

Comment:
```suggestion
    "pynrrd~=0.4"
```

Using `>=` means `pynrrd==2.0` would also be valid. I know that the rest of the requirements use an incorrect notation, sue me 😇